api/.env.defaults

```diff
@@ -8,7 +8,7 @@ NODE_CONFIG_ENV=development
 ## SECRETS - Note: these values need to be faked so that tests pass in the pipeline
 #IDAM_SECRET=AAA
 #S2S_SECRET=AAA
-APPINSIGHTS_INSTRUMENTATIONKEY=AAA
+APPINSIGHTS_INSTRUMENTATIONKEY=5f607ccb-4ab5-4c9d-a4b9-801868bdf698
```

Comment:
This should be reverted before any merge

Reply:
yes sure @danlysiak